src/renderer/models/account.ts

```diff
@@ -0,0 +1,140 @@
+import {Keypair} from "@chainsafe/bls/lib/keypair";
+import {readdirSync} from "fs";
+import {ICGKeystoreFactory} from "../services/interfaces";
+import {Eth1CGKeystoreFactory} from "../services/Eth1CGKeystore";
+
+export interface IAccount {
+    name: string;
+    directory: string;
+    sendStats: boolean;
+}
+
+export class CGAccount implements IAccount {
+    public name: string;
+    public directory: string;
+    public sendStats: boolean;
+
+    private validators: Keypair[] = [];
+    private keystoreTarget: ICGKeystoreFactory;
+
+    public constructor(account: IAccount, keystoreTarget: ICGKeystoreFactory = Eth1CGKeystoreFactory) {
+        this.name = name;
+        // Add / to the end if not provided
+        this.directory = account.directory + (account.directory.endsWith("/") ? "" : "/");
+        this.sendStats = account.sendStats;
+        this.keystoreTarget = keystoreTarget;
+    }
+
+    /**
+     * should return addresses from validator keystores in account directory
+     */
+    public getValidatorsAddresses(): string[] {
+        // Loop trough files in account directory
+        const keystoreFiles: string[] = this.getKeystoreFiles();
+
+        const validatorAddresses: string[] = keystoreFiles
+            .map(file => new this.keystoreTarget(file))
```

Comment:
I think this can throw exception if user puts invalid keystore in app directory (manual import) so you should filter our keystores tha throws expetion. It probably makes sense and it would reduce code duplication that `getKEystoreFiles` returns keystore instances (already filtered)